Make copyright header lines uniform

isle2983 opened the pull request:
Fixes a few instances of copyright headers deviating from the norm as
discovered using the `copyright_header_report.py` script that was introduced in the previous PR

fanquake commented:
Please squash these into a single commit.

dcousens commented:
trivial ACK

MarcoFalke commented:
Please squash your commits according to https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits

isle2983 commented:
I have squashed the three into one commit.

MarcoFalke commented on src/univalue/test/unitester.cpp:
```diff
@@ -1,4 +1,4 @@
-// Copyright 2014 BitPay Inc.
```
univalue is a subtree. You'd need to change it upstream.

isle2983 commented:
Thanks @MarcoFalke. I have dropped the changes from src/univalue/* here and I have submitted them as a PR to @jgarzik's main univalue repo.

fanquake commented:
ACK b0cc47d

MarcoFalke commented:
ACK b0cc47d

paveljanik commented:
Needs rebase

fanquake commented:
@isle2983 Can you rebase this so it can be merged?

isle2983 committed:
[copyright] copyright header style uniform

Three categories of modifications:

1)

1 instance of 'The Bitcoin Core developers \n',
1 instance of 'the Bitcoin Core developers\n',
3 instances of 'Bitcoin Core Developers\n', and
12 instances of 'The Bitcoin developers\n'

are made uniform with the 443 instances of 'The Bitcoin Core developers\n'

2)

3 instances of 'BitPay, Inc\.\n' are made uniform with the other 6
instances of 'BitPay Inc\.\n'

3)

4 instances where there was no '(c)' between the 'Copyright' and the year
where it deviates from the style of the local directory.

isle2983 commented:
rebased and conflicts resolved in four files

src/policy/policy.cpp - The conflict was with date being bumped to '2016' by a prior commit on the same line as this pull's edit.

src/policy/policy.h - same as with policy.cpp

src/test/bctest.py - The conflict was with a new 'The Bitcoin Core developers' copyright holder was added in the adjacent line to my edit where the comma was removed after 'BitPay'.

src/test/bitcoin-util-test.py - same as with bctest.py
